src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -26,6 +28,7 @@ import { TutorialGroupsConfigurationService } from 'app/course/tutorial-groups/s
     selector: 'jhi-course-overview',
     templateUrl: './course-overview.component.html',
     styleUrls: ['course-overview.scss', './tab-bar/tab-bar.scss'],
+    providers: [MetisConversationService, MetisService],
```

Comment:
I am not sure if it would be correct to have the course component provide the MetisConversationService. Can an Angular expert confirm this is a reasonable approach and does not cause any performance reduction?

Reply:
This was also discussed with @stefanwaldhauser. And this approach is correct one as caching will work better. @stefanwaldhauser can you confirm this?

Reply:
There was an angular example (can't find it anymore. Something hero cache) where they used service from a component for caching so this should be reasonable approach.